Tools/PowershellModule/doc/new-private-repository-azure.md

```diff
@@ -1,238 +1,226 @@
-# Windows Package Manager REST source
+# Create a rest source Windows Package Manager repository
 
-The Windows Package Manager provides a distribution channel for software packages containing their tools and applications. The instructions contained within this document provide guidance on how to setup a REST source that can be connected to using the Windows Package Manager. Providing a comprehensive package manager solution that consists of a command line tool and a set of services for installing applications on Windows 10.
+This section provides guidance on how to create a rest source repository that can be connected to using the Windows Package Manager for ISVs or Publishers who may have additional privacy requirements for their software distribution.
```

Comment:
```suggestion
This section provides guidance on how to create a REST source repository that can be connected to using the Windows Package Manager for ISVs or Publishers who may have additional privacy requirements for their software distribution.
```

Reply:
As discussed offline, entire block needs editing

Reply:
This block doesn't appear to have been revised.